src/odemis/util/transform.py

```diff
@@ -193,6 +193,70 @@ def _optimal_rotation(x, y):
     return numpy.dot(V.T, U.T)
 
 
+def to_physical_space(ji, shape, pixel_size=None):
+    """
+    Converts an image index into a coordinate in physical space.
+
+    Images are displayed on a computer screen with the origin in the top-left
+    corner. The column-index `i` is used as the primary axis and the row-index
+    `j` is used as the secondary axis. This represents a left-handed coordinate
+    system. Physical coordinates are typically described in a right-handed
+    coordinate system.
+
+    Calculations should be done preferably in the right-handed physical
+    coordinate system. Calculations using mixed left-handed and right-handed
+    coordinates should be avoided.
+
+    `to_physical_space` converts an image index into a coordinate in physical
+    space. The sign of the row-index is changed such that is increasing in the
+    upward direction, the origin is moved to the center of the image, and the
+    `x`-axis is made the primary axis.
+
+
+    Parameters
+    ----------
+    ji : tuple, list of tuples, ndarray
+        Pixel index, or list of indices, into a 2-dimensional array.
```

Comment:
Let `A` be a `n`-by-`m` matrix. Then `A[j, i]` is the element of `A` at row `j` and column `i`. It is also possible to use a tuple of indices: `A[(j, i)]`.

The order `j`, `i` is such that it can be directly used as in index into a Numpy array without having to unpack or reverse the tuple.

It might make more sense to use `i` as the row index and `j` as the column index; but I thought this had your preference.

Reply:
As discussed we will use the order `ji`. 

I still suggest to add here again, that `i is primary` and `j secondary axis` and that `pixel indices are in image space` in case someone looks up the parameter, but does not read the full doc string.